docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+
+1. Clone OpenVINO repository and init submodules:
+   ```sh
+   git clone https://github.com/openvinotoolkit/openvino.git
+   cd openvino
+   git submodule update --init --recursive
+   ```
+
+2. Install build dependencies using the `install_build_dependencies.sh` script in the
+   project root folder.
+   ```sh
+   sudo ./install_build_dependencies.sh
+   ```
+
+3. Create a build folder:
+   ```sh
+   mkdir build && cd build
+   ``` 
+
+4. To cross compile OpenVINO Runtime for RISC-V devices, run `cmake` with specified `CMAKE_TOOLCHAIN_FILE` and `RISCV_TOOLCHAIN_ROOT`:
+   ```sh
+   cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_INSTALL_PREFIX=<openvino_install_path> \
+     -DENABLE_INTEL_CPU=ON \
+     -DENABLE_INTEL_GPU=OFF \
+     -DENABLE_INTEL_NPU=OFF \
+     -DENABLE_AUTO=ON \
+     -DENABLE_HETERO=ON \
+     -DENABLE_MULTI=ON \
+     -DENABLE_JS=OFF \
+     -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/<toolchain_file> \
+     -DRISCV_TOOLCHAIN_ROOT=<xuantie_install_path> \
+     -DENABLE_TESTS=ON \
+     -DENABLE_SAMPLES=ON
+   ```
+   > **NOTE**: To build OpenVINO Runtime for different versions of RVV, you just need to specify corresponding toolchain files. For exmaple, you can replace `<toolchain_file>` with `riscv64-071-thead-gnu.toolchain.cmake` for RVV 0.7.1 and `riscv64-100-thead-gnu.toolchain.cmake` for RVV 1.0 respectively.
+
+   Then run `make` to build the project:
+   ```sh
+   make install -j$(nproc)
+   ```
+
+### Build without RVV
+0. Prerequisite - build `riscv-gnu-toolchain`:
+   ```sh
+   git clone https://github.com/riscv-collab/riscv-gnu-toolchain.git
+   cd riscv-gnu-toolchain
+   ./configure --prefix=/opt/riscv
+   make linux build-qemu -j$(nproc)
+   ```
+   > **NOTE**: The `build-qemu` target is optional here as well. More information can be seen [here](https://github.com/riscv-collab/riscv-gnu-toolchain).
+
+1. Clone OpenVINO repository and init submodules:
+   ```sh
+   git clone https://github.com/openvinotoolkit/openvino.git
+   cd openvino
+   git submodule update --init --recursive
+   ```
+
+2. Install build dependencies using the `install_build_dependencies.sh` script in the
+   project root folder.
+   ```sh
+   sudo ./install_build_dependencies.sh
+   ```
+
+3. Create a build folder:
+   ```sh
+   mkdir build && cd build
+   ``` 
+
+4. To cross compile OpenVINO Runtime for RISC-V devices, run `cmake` with specified `CMAKE_TOOLCHAIN_FILE` and `RISCV_TOOLCHAIN_ROOT`:
+   ```sh
+   cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_INSTALL_PREFIX=<openvino_install_path> \
+     -DENABLE_INTEL_CPU=ON \
+     -DENABLE_INTEL_GPU=OFF \
+     -DENABLE_INTEL_NPU=OFF \
+     -DENABLE_AUTO=ON \
+     -DENABLE_HETERO=ON \
+     -DENABLE_MULTI=ON \
+     -DENABLE_JS=OFF \
+     -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/riscv64-071-thead-gnu.toolchain.cmake \
```

Comment:
This cmake toolchain file is used only for RVV0.7.1. Why is it mentioned in the section `Build without RVV`?
Can we reuse the existing default [toolchain](https://github.com/openvinotoolkit/openvino/blob/239466ca5d5333af11ee750260215941737383b3/cmake/toolchains/riscv64-gnu.toolchain.cmake)? What do you think?

Reply:
Yes, we should use the general toolchain file here. Thank you!